packages/cache-common/src/types/Cache.ts

```diff
@@ -24,4 +24,9 @@ export type Cache = {
    * Clears the cache.
    */
   readonly clear: () => Readonly<Promise<void>>;
+
+  /**
+   * Retrieve the entries currently in cache.
+   */
+  readonly entries: () => Readonly<Promise<unknown>>;
```

Comment:
Seems you forgot the `entries` implementation in `packages/cache-common/src/createFallbackableCache.ts` and `packages/cache-common/src/createNullCache.ts`